CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
```

Comment:
@vladanost Let's link to the github issues we close as well in changelogs. This is one I know: https://github.com/bigcommerce/bigcommerce-for-wordpress/issues/244